config/api.yml

```diff
@@ -750,13 +750,17 @@
         :identifier: cloud_volume_show_list
       - :name: delete
         :identifier: cloud_volume_delete
+      - :name: delete
+        :identifier: cloud_volume_safe_delete
     :resource_actions:
       :get:
       - :name: read
         :identifier: cloud_volume_show
       :post:
       - :name: delete
         :identifier: cloud_volume_delete
+      - :name: delete
```

Comment:
are we exposing both types of delete ?      if so, maybe renaming second one to safe_delete

Reply:
I also think that the name here need to be safe_delete

Reply:
Yeah, I was copy-pasting too fast (sorry)